From: martin f. krafft <madduck@debian.org>
Subject: [PATCH] add DESTDIR support to Makefile

This is needed for distros to be able to install files to a temporary root
location, without affecting the PREFIX paths that are used at runtime.

Signed-off-by: martin f. krafft <madduck@debian.org>

---
 Makefile |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/Makefile b/Makefile
index 6eade1e..bafec26 100644
--- a/Makefile
+++ b/Makefile
@@ -31,14 +31,14 @@ $(help_out): README
 	./create-help.sh $$CMD
 
 install:: all
-	install -d -m 755 "$(bindir)"
-	install tg "$(bindir)"
-	install -d -m 755 "$(cmddir)"
-	install $(commands_out) "$(cmddir)"
-	install -d -m 755 "$(hooksdir)"
-	install $(hooks_out) "$(hooksdir)"
-	install -d -m 755 "$(sharedir)"
-	install -m 644 $(help_out) "$(sharedir)"
+	install -d -m 755 "$(DESTDIR)$(bindir)"
+	install tg "$(DESTDIR)$(bindir)"
+	install -d -m 755 "$(DESTDIR)$(cmddir)"
+	install $(commands_out) "$(DESTDIR)$(cmddir)"
+	install -d -m 755 "$(DESTDIR)$(hooksdir)"
+	install $(hooks_out) "$(DESTDIR)$(hooksdir)"
+	install -d -m 755 "$(DESTDIR)$(sharedir)"
+	install -m 644 $(help_out) "$(DESTDIR)$(sharedir)"
 
 clean::
 	rm -f tg $(commands_out) $(hooks_out) $(help_out)
-- 
tg: (11a91e4..) fixes/destdir (depends on: upstream)
